rest_framework/fields.py

```diff
@@ -1042,10 +1042,16 @@ def to_internal_value(self, data):
         except KeyError:
             self.fail('invalid_choice', input=data)
 
+    def representation_value(self, value):
+        try:
+            return self.choice_strings_to_values[six.text_type(value)]
+        except KeyError:
+            return value
```

Comment:
Can't you just do `if six.text_type(value) in self.choice_strings_to_values` instead of relying on a try/except here?